src/rdf4cpp/rdf/Literal.cpp

```diff
@@ -1319,13 +1317,25 @@ Literal Literal::regex_replace(Literal const &pattern, Literal const &replacemen
     return this->regex_replace(*repl, node_storage);
 }
 
-util::TriBool Literal::contains(std::string_view const needle) const noexcept {
+util::TriBool Literal::contains_normalized(std::string_view const needle) const noexcept {
```

Comment:
This is still an open question, just highlighting this for later
- [ ] figure out if we actually want to normalize the inputs